index.js

```diff
@@ -0,0 +1,196 @@
+     "featured":"../Assets/Images/header-illsutration-mobile@2x.png", 
+     "technologies":["Ruby on rails","css","javascript","html"],
+     "live":"ldouglasot.github.io",
+     "source":"https://github.com/LDouglasOT/ldouglasot.github.io"
+  },{
+    "name":"Multi-Post Stories</span><span>Gain+Glory",
+     "description":"This is a website developed using html,css and javascript following best cl/ci practices", 
+     "featured":"../Assets/Images/header-illsutration-mobile@2x.png", 
+     "technologies":["Ruby on rails","css","javascript","html"],
+     "live":"ldouglasot.github.io",
+     "source":"https://github.com/LDouglasOT/ldouglasot.github.io"
+  },{
+    "name":"Multi-Post Stories</span><span>Gain+Glory",
+     "description":"This is a website developed using html,css and javascript following best cl/ci practices", 
+     "featured":"../Assets/Images/header-illsutration-mobile@2x.png", 
+     "technologies":["Ruby on rails","css","javascript","html"],
+     "live":"ldouglasot.github.io",
+     "source":"https://github.com/LDouglasOT/ldouglasot.github.io"
+  },{
+    "name":"Multi-Post Stories</span><span>Gain+Glory",
+     "description":"This is a website developed using html,css and javascript following best cl/ci practices", 
+     "featured":"../Assets/Images/header-illsutration-mobile@2x.png", 
+     "technologies":["Ruby on rails","css","javascript","html"],
+     "live":"ldouglasot.github.io",
+     "source":"https://github.com/LDouglasOT/ldouglasot.github.io"
+  },
+
+]
+
+window.addEventListener('load', ()=>{
+
+  const untitlelist=document.querySelector('.works')
+
+  const li=document.createElement('ul')
+  li.classList.add('works-ul')
+  const insertitems=(name,description,featured,live,source,technologies)=>{
+  return `
+
+  <div class="top-notch">
+    <img class="imagedescription" src="https://wallpaperaccess.com/full/220746.jpg" alt='capstone' />
+    <div class="top-notch-content">
+      <h3 class="top-tag">
+        <span>${name}</span>
+      </h3>
+      <ul class="top-notch-content-ul">
+      
+        <li class="top-notch-content-ul-li">${technologies[0]}</li>
+        <li class="top-notch-content-ul-li">${technologies[1]}</li>
+        <li class="top-notch-content-ul-li">${technologies[2]}</li>
+        <li class="top-notch-content-ul-li">${technologies[3]}</li>
+      </ul>
+      <br />
+      <button class="top-notch-content-a" id='seeproject'>See Project</b>
+      <br />
+    </div>
+  </div>
+ 
+  `
+   }
+   projectlist.map((item)=>{
+    const lists=document.createElement('li')
+    lists.classList='list-item'
+    lists.innerHTML=insertitems(item.name, item.description, item.featured, item.live, item.source,item.technologies)
+    li.appendChild(lists)
+   })
+   untitlelist.append(li)
+});
+
+
+
+window.onload = function() {
+if (window.matchMedia("(max-width: 680px)").matches) {
+const btnclick=document.querySelector('#seeproject')
+console.log(btnclick)
```

Comment:
[OPTIONAL] Kindly take all the `console.log`s from your code since they are for testing purposes only and it is not good practice to leave them in the final project. :pray: